web/src/app/administration/classification/details/classification-details.component.ts

```diff
@@ -128,7 +126,7 @@ export class ClassificationDetailsComponent implements OnInit, OnDestroy {
         // TSK-891 fix: The property is already set and is crucial value
         // Wrapped with an if to set a default if not already set.
         if (!this.classification.category) {
-          this.classification.category = categories[0];
+          [this.classification.category] = categories;
```

Comment:
haha destruct operator for arrays ... very tricky thing for non js devs ;)

Reply:
Yes, it is - i googled twice! ;)